src/draw/nxp/pxp/lv_pxp_osa.c

```diff
@@ -57,12 +62,19 @@ static void _pxp_run(void);
  */
 static void _pxp_wait(void);
 
+#if defined(__ZEPHYR__)
+    /**
+    * Interrupt handler for Zephyr IRQ
+    */
+    static void _pxp_zephyr_irq_handler(void *);
+#endif
+
 /**********************
  *  STATIC VARIABLES
  **********************/
 
-#if defined(SDK_OS_FREE_RTOS)
-    static SemaphoreHandle_t xPXPIdleSemaphore;
+#if LV_USE_OS
+    static lv_thread_sync_t pxp_sync;
 #endif
 static volatile bool ucPXPIdle;
```

Comment:
As this file no longer refer to FreeRTOS only, please rename this variable to "pxp_idle". Has nothing to do directly with your changes but please include this change as well - if not big deal. Thanks.